tests/install/openqa_webui.pm

```diff
@@ -53,7 +53,7 @@ EOF
 }
 
 sub install_containers {
-    assert_script_run("zypper --non-interactive install docker git", timeout => 600);
+    assert_script_run("zypper --non-interactive install docker git podman", timeout => 600);
```

Comment:
Have you checked whether we still use docker elsewhere? If no, I suppose docker can be removed from here.

Reply:
Yes, and I'm proposing #85 separately.